drivers/sensor/stts751/stts751.h

```diff
@@ -20,21 +20,17 @@
 #include "stts751_reg.h"
 
 struct stts751_config {
-	char *master_dev_name;
+	struct i2c_dt_spec i2c;
```

Comment:
I think this should be inside a conditional `#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)` like `i2c_slv_addr` was before this patch.

Reply:
There's an `#error` if `#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)` is false. Looks to me as if the `#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)`'s could be removed all together?

Reply:
I've added the conditional now!